snuba/admin/views.py

```diff
@@ -472,6 +481,8 @@ def clickhouse_trace_query() -> Response:
                     int(query_trace_data.port),
                     storage,
                     sql,
+                    False,
+                    g.user,
```

Comment:
Ditto here about the naming of `g.user`

Reply:
See https://github.com/getsentry/snuba/pull/6251#discussion_r1736826390